physlr/physlr.py

```diff
@@ -37,6 +38,7 @@ def progress_bar_for_file(fin):
         mininterval=1, smoothing=0.1,
         bar_format="{percentage:4.1f}% {elapsed} ETA {remaining} {bar}")
 
+
```

Comment:
Only one blank line between functions please.

Reply:
Oh okay, pycharm reports warning for one blank line and asks for 2. I'll change it.